Build failure: pkgsStatic.util-linux

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

nix-build -A pkgsStatic.util-linux

### Can Hydra reproduce this build failure?

No, Hydra cannot reproduce this build failure.

### Link to Hydra build job

_No response_

### Relevant log output

```console
> libtool:   error: cannot build a shared library See the libtool documentation for more information. Fatal configuration error.
       > make[2]: *** [Makefile:8284: pam_lastlog2.la] Error 1
       > make[2]: *** Waiting for unfinished jobs....
       > make[2]: Leaving directory '/build/util-linux-2.40.4'
       > make[1]: *** [Makefile:16952: all-recursive] Error 1
       > make[1]: Leaving directory '/build/util-linux-2.40.4'
       > make: *** [Makefile:7294: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.13.4, NixOS, 25.05 (Warbler), 25.05.20250226.5135c59`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.12`
 - nixpkgs: `/nix/store/vdyq3cl640n39sclkc8br4xgd09fdj2w-source`

### Notify maintainers


---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/ut/util-linux/package.nix
+++ b/pkgs/by-name/ut/util-linux/package.nix
@@ -105,6 +105,10 @@ stdenv.mkDerivation rec {
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       # Doesn't build on Darwin, also doesn't really make sense on Darwin
       "--disable-liblastlog2"
+    ]
+    ++ lib.optionals stdenv.hostPlatform.isStatic [
+      # Mandatory shared library.
+      "--disable-pam-lastlog2"
     ];
 
   makeFlags = [

